tabs/ansible/cassandra/blueprint.yaml

```diff
@@ -124,7 +124,7 @@ outputs:
   cassandra_port:
     description: Postgres Port
     value: 9042
-
+    
```

Comment:
```suggestion

```

Reply:
Please look at the row file, its properly formatted now.